Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -759,7 +867,7 @@ void GameListCtrl::OnKeyPress(wxListEvent& event)
   static int lastKey = 0, sLoop = 0;
   int Loop = 0;
 
-  for (int i = 0; i < (int)m_ISOFiles.size(); i++)
+  for (int i = 0; i < (int)m_shown_files.size(); i++)
```

Comment:
Moar c-style casts. Although this one looks a little like it could be rewritten as something else (`std::find_first`?) if the order of `m_shown_files` is the same as the ordering inside the game list; and the `GameListItem` has the ability to return the same text that `bleh` would have later. Might be wrong tho...